srcpkgs/widelands/template

```diff
@@ -1,21 +1,23 @@
 # Template file for 'widelands'
 pkgname=widelands
-version=21
-revision=5
-wrksrc="${pkgname}-build${version}"
+# Widelands 1.0 release supercedes beta builds
+reverts="21_5"
```

Comment:
this is missing many versions (and revisions) that were packaged

Reply:
Should it revert every packaged version?

Reply:
Think I got them all.